back/src/controllers/loginController.ts

```diff
@@ -1,43 +1,36 @@
 import Koa from 'koa'
 import jwt, { Secret } from 'jsonwebtoken'
-import { PrismaClient } from '@prisma/client'
+import { prisma } from '../prisma/client'
 import { checkPassword } from '../helpers/passwordHash'
 
-const prisma = new PrismaClient()
-
 export const loginController = async (ctx: Koa.Context) => {
   const { dni, password } = ctx.request.body
   const dniUpperCase = dni.toUpperCase()
 
-  try {
-    const user = await prisma.user.findUnique({
-      where: { dni: dniUpperCase as string },
-      select: { id: true, password: true }
-    })
+  const user = await prisma.user.findUnique({
+    where: { dni: dniUpperCase as string },
+    select: { id: true, password: true },
+  })
 
-    if (!user) {
-      ctx.status = 401
-      ctx.body = { error: 'User not found' }
-      return
-    }
+  if (!user) {
+    ctx.status = 401
+    ctx.body = { error: 'User not found' }
+    return
+  }
 
-    const isPasswordValid = await checkPassword(password, user.password)
+  const isPasswordValid = await checkPassword(password, user.password)
 
-    if (!isPasswordValid) {
-      ctx.status = 401
-      ctx.body = { error: 'Invalid password' }
-      return
-    }
+  if (!isPasswordValid) {
+    ctx.status = 401
```

Comment:
422: https://stackoverflow.com/questions/3050518/what-http-status-response-code-should-i-use-if-the-request-is-missing-a-required